src/engine.cpp

```diff
@@ -538,10 +538,24 @@ class HangulState : public InputContextProperty {
     LookupMethod lastLookupMethod_;
 };
 
+HanjaTable *load_table() {
+    if constexpr (isAndroid()) {
```

Comment:
This actually also makes some sense on non android, can we do it like:
1. locate the file
2. load the table with file is file is ok.
3. if empty, or table load is failed, call table_load with nullptr